exo-create/helpers/helpers.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+func createProjectJSON(templateDir string) error {
+	content := "{\n\"AppName\": \"my-app\",\n\"ExocomVersion\": \"0.22.1\",\n\"AppVersion\": \"0.0.1\",\n\"AppDescription\": \"\"\n}"
```

Comment:
This can be defined outside the func so we can use actual newlines to make it more readable
```go
var projectJsonContent = `
{
  "AppName": "my-app",
  "ExocomVersion": "0.22.1",
  "AppVersion": "0.0.1",
  "AppDescription": ""
}
`
```
Same for the yml content.